docs/extend/extend-apiml/api-mediation-onboard-an-existing-java-rest-api-service-without-spring-boot-with-zowe-api-mediation-layer.md

```diff
@@ -256,6 +256,10 @@ After you add API Mediation Layer integration endpoints, you are ready to add se
     ```yaml
     serviceId: hellospring
     title: HelloWorld Spring REST API
+    eureka:
```

Comment:
Remove this part. This wasn't added by me and we would need to test it.

Reply:
Removed